src/java/com/zimbra/cs/nginx/NginxLookupExtension.java

```diff
@@ -1153,6 +1153,10 @@ private void useDomainExternalRoutingWhenNoUserFound(NginxLookupRequest req, ILd
             DomainExternalRouteInfo domain =
                     getDomainExternalRouteInfo(zlc, config, authUserWithRealDomainName);
             if (domain == null || !domain.useExternalRouteIfAccountNotExist()) {
+                if (LC.zimbra_additional_logging.booleanValue()){
+                    ZimbraLog.security.info("user not found: " + authUserWithRealDomainName+"; oip="+ req.clientIp);
+                    ZimbraLog.nginxlookup.info("Error occurred during authentication: account not found for [%s]. oip=%s ",authUserWithRealDomainName,req.clientIp);
```

Comment:
```suggestion
                    ZimbraLog.nginxlookup.info("Error occurred during authentication: account not found for [%s]. oip=%s ", authUserWithRealDomainName, req.clientIp);
```
Can we use account.info logger here.
Also on zm-sync-store relate PR, same type of format of logging should be done